### PR TITLE
Add rendering tests

### PR DESCRIPTION
**Pull request by SheepFromHeaven**

### CODE REVIEW

File: tests/help.js [hunk 1]
```diff
@@ -1,14 +1,8 @@
 const chai = require('chai');
 const sinonChai = require('sinon-chai');
-const { JSDOM } = require('jsdom');
```
Comment (tiagomapmarques) [inline tests/help.js]: do we still need JsDOM or can we remove the dependency?

File: package.json [hunk 2]
```diff
@@ -38,6 +39,7 @@
     "@types/mocha": "~5.2",
     "@types/sinon": "~5.0",
     "@types/sinon-chai": "~3.2",
+    "basichtml": "^0.20.0",
```
Comment (tiagomapmarques) [inline package.json]: can we write this as `~0.20`?

File: tests/help.js (hunk 1 above)
Comment (jurekbarth) [reply tests/help.js]: Ah yes 👍

File: package.json (hunk 2 above)
Comment (jurekbarth) [reply package.json]: Sure :)

**Pull request merged**